sretoolbox/container/skopeo.py

```diff
@@ -67,7 +73,8 @@ def inspect(self, image, creds=None):
     def _run_skopeo(self, subcomand, *args,
                     src_creds=None,
                     dest_creds=None,
-                    creds=None):
+                    creds=None,
+                    all_=False):
```

Comment:
Just out of curiosity... why the trailing underscore in all_? :)

Reply:
to avoid clashing with https://docs.python.org/3/library/functions.html#all

Reply:
From pep8: `single_trailing_underscore_: used by convention to avoid conflicts with Python keyword`

Reply:
awesome, thx!